src/arrayReverse.test.js

```diff
@@ -3,18 +3,66 @@
 describe(`Function 'arrayReverse':`, () => {
   const arrayReverse = require('./arrayReverse');
 
-  it(`should be declared`, () => {
-    expect(arrayReverse).toBeInstanceOf(Function);
+  it(`should be declared`, () => expect(arrayReverse).toBeDefined());
+
+  it(`should return an array of strings`, () => {
+    const result = arrayReverse(['One', 'two']);
+
+    expect(result).toBeInstanceOf(Array);
+    result.forEach((word) => expect(typeof word).toBe('string'));
   });
 
-  it(`should return an array`, () => {
+  it(`should return an array of the same length`, () => {
+    const array = ['One', 'two', 'three'];
+
+    expect(arrayReverse(array).length).toBe(array.length);
+  });
 
+  it(`should return a reversed array containing one word`, () => {
+    expect(arrayReverse(['Hello'])).toEqual(['olleH']);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
+  it(`should return a reversed array containing multiple words`, () => {
     expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
```

Comment:
The expected result for reversing the array `['Mate', 'Academy']` should be `['yM', 'edacAetaM']` according to the task description, which specifies reversing both the strings and their order. Please verify the expected output.